packages/toggle/README.md

```diff
@@ -23,49 +23,56 @@ npm install @leafygreen-ui/toggle
+| Prop        | Type                               | Description                                                                                                                                                                                                     | Default     |
+| ----------- | ---------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| `darkMode`  | `boolean`                          | Determines if the Toggle will render the dark mode styles.                                                                                                                                                      | `false`     |
+| `size`      | `'default'`, `'small'`, `'xsmall'` | Sets the size of the toggle.                                                                                                                                                                                    | `'default'` |
+| `checked`   | `boolean`                          | Set's the checked state of the Toggle. We recommend not passing this prop, and instead using the `onChange` event to sync your application's state in most cases.                                               |             |
+| `disabled`  | `boolean`                          | Disables the Toggle.                                                                                                                                                                                            | `false`     |
+| `onClick`   | `(MouseEvent) => void`             | The event handler function for the 'onchange' event. Receives the associated `event` object as the first argument.                                                                                              |             |
+| `onChange`  | `(checked, MouseEvent) => void`    | `onChange` fires when the internally-managed `checked` state of the component is updated. Receives the updated checked state of the toggle as its first argument, and the associated mouse event as the second. |             |
+| `className` | `string`                           | Adds a className to the outermost element.                                                                                                                                                                      | `''`        |
+| ...         | HTML `button` attributes           | Any natively-supported HTML button attributes will be applied to the `button` element                                                                                                                           |             |
+
+## Accessibility
```

Comment:
Would it make sense to have a useAccessibleToggle hook which takes in a Label component and a Toggle component and returns cloned versions of the components that are accessibly associated with one another? Might not be necessary for this version, but also might help us ensure that our Toggle components are being implemented accessibly and reduce work to do so on the consuming applications' end.

Reply:
Maybe, but I'd doubt it'd be much easier to get people to use it tbh.

Reply:
Honestly, I'd rather educate people a bit on this stuff than us providing abstractions for actions they themselves have to take. It can help the company as a whole understand accessibility a bit better.

Reply:
Another thought: a hook that accepts the label and toggle props and returns cloned (accessible) props to be spread on the respective elements... while I am all for increasing education/awareness of accessibility best-practices, I also see it as our goal to make these things as baked in as possible, and our components as easy to use accessibly as possible. Again, this doesn't have to happen in this PR but just something I'm thinking about

Reply:
I think where I'm not convinced is that I feel that having an external hook isn't baking in the behavior at all. It's effectively as much work to make the component accessible, but stripping any education about accessibility away in the process. If we made a hook, there's still something to learn, but what the developer would learn in that case isn't transferrable to other parts of their work.

 I generally favor standard solutions over proprietary ones where possible, and this is one of those cases.